EIPS/eip-6224.md

```diff
@@ -204,7 +221,7 @@ There are a few design decisions that have to be specified explicitly:
 
 #### Usage
 
-The extensions of this EIP SHOULD add proper access control checks to the described non-view methods. 
+The extensions of this EIP SHOULD add proper access control checks to the described non-view methods.
 
 The `getContract` and `getImplementation` methods MUST revert if the nonexistent contracts are queried.
```

Comment:
This paragraph, and the following one, introduce new requirements. The rationale section should only explain choices made within the proposal, while any requirements have to be defined in the specification section. In other words, `SHOULD`, `MAY`, `MUST`, etc. can't appear in the rationale.